e2e/refund/refund.spec.ts

```diff
@@ -0,0 +1,170 @@
+import { Page, expect, request, test } from "@playwright/test";
+import fs from "fs";
+import path from "path";
+
+import dict from "../../src/i18n/i18n";
+import { UTXO } from "../../src/utils/blockchain";
+import {
+    decodeLiquidRawTransaction,
+    elementsSendToAddress,
+    generateLiquidBlock,
+    getBolt12Offer,
+    getLiquidAddress,
+    setFailedToPay,
+} from "../utils";
+
+const setupSwapAssets = async (page: Page) => {
+    await page.locator(".arrow-down").first().click();
+    await page.getByTestId("select-L-BTC").click();
+    await page
+        .locator(
+            "div:nth-child(3) > .asset-wrap > .asset > .asset-selection > .arrow-down",
+        )
+        .click();
+    await page.getByTestId("select-LN").click();
+};
+
+const fillSwapDetails = async (page: Page) => {
+    await page.getByTestId("invoice").fill(await getBolt12Offer());
+    await page.getByTestId("sendAmount").fill("0.005");
+    await page.getByTestId("create-swap-button").click();
+};
+
+const createAndVerifySwap = async (page: Page, rescueFile: string) => {
+    await page.goto("/");
+    await setupSwapAssets(page);
+    await fillSwapDetails(page);
+
+    const downloadPromise = page.waitForEvent("download");
+    await page.getByRole("button", { name: dict.en.download_new_key }).click();
+    await (await downloadPromise).saveAs(rescueFile);
+
+    await page.getByTestId("rescueFileUpload").setInputFiles(rescueFile);
+    await page.getByText("address").click();
+};
```

Comment:
This is copy pasted from here: https://github.com/BoltzExchange/boltz-web-app/blob/main/e2e/refund/rescueFile.spec.ts#L16
Let's move that logic into some utils file and import it from there